XenAdmin/Wizards/NewSRWizard_Pages/SrWizardHelpers.cs

```diff
@@ -40,6 +40,21 @@ namespace XenAdmin.Wizards.NewSRWizard_Pages
 {
     public class SrWizardHelpers
     {
+        public static bool ValidateServerAddress(String address)
+        {
+            var serverNameType = Uri.CheckHostName(address);
+            if (serverNameType == UriHostNameType.Unknown || serverNameType == UriHostNameType.IPv6)

```

Comment:
When the then- and the else-statement is just a return true/false, it's usually better to just return the condition itself. I'm fine with this now, but it'd be nicer to have a one-liner here.